core/src/components/radio-group/radio-group.tsx

```diff
@@ -224,13 +240,69 @@ export class RadioGroup implements ComponentInterface {
     radioToFocus?.setFocus();
   }
 
+  /**
+   * Renders the helper text or error text values
+   */
+  private renderHintText() {
+    const { helperText, errorText, helperTextId, errorTextId } = this;
+
+    const hasHintText = !!helperText || !!errorText;
+    if (!hasHintText) {
+      return;
+    }
+
+    return (
+      <div class="radio-group-top">
+        <div id={helperTextId} class="helper-text">
+          {helperText}
+        </div>
+        <div id={errorTextId} class="error-text">
+          {errorText}
+        </div>
+      </div>
+    );
+  }
+
+  private getHintTextID(): string | undefined {
```

Comment:
It is a bit confusing that we alternate between `hint` and `supporting` text when referring to helper and error text collectively. However, since this is internal, I didn't make any changes for now. We can update it across all components in the future if needed.